lib/go-tc/deliveryservice_servers.go

```diff
@@ -19,21 +21,47 @@ import "time"
 
 // DeliveryServiceServerResponse ...
 type DeliveryServiceServerResponse struct {
+	Orderby  string                  `json:"orderby"`
 	Response []DeliveryServiceServer `json:"response"`
 	Size     int                     `json:"size"`
-	OrderBy  string                  `json:"orderby"`
 	Limit    int                     `json:"limit"`
 }
 
+type DSSMapResponse struct {
+	DsId    int   `json:"dsId"`
+	Replace bool  `json:"replace"`
+	Servers []int `json:"servers"`
+}
+
+type DSSReplaceResponse struct {
+	Alerts   []Alert        `json:"alerts"`
+	Response DSSMapResponse `json:"response"`
+}
+
+type DSServersResponse struct {
+	Response DeliveryServiceServers `json:"response"`
+}
+
+type DeliveryServiceServers struct {
+	ServerNames []string `json:"serverNames"`
+	XmlId       string   `json:"xmlId"`
+}
+
 // DeliveryServiceServer ...
 type DeliveryServiceServer struct {
-	Server          *int             `json:"server"`
-	DeliveryService *int             `json:"deliveryService"`
-	LastUpdated     *TimeNoMod       `json:"lastUpdated" db:"last_updated"`
+	Server          *int       `json:"server" db:"server"`
+	DeliveryService *int       `json:"deliveryService" db:"deliveryservice"`
+	LastUpdated     *TimeNoMod `json:"lastUpdated" db:"last_updated"`
 }
 
+type Filter int
+const (
+	ASSIGNED Filter = iota
```

Comment:
Go enums and constants are CamelCased, not ALLCAPS. These should be `Assigned`, `Unassigned`, `Eligible`